apps/server-admin/src/modules/event/EventMutations.ts

```diff
@@ -1,9 +1,9 @@
-import { AddUserInEventMutation } from './mutations/AddUserInEventMutation';
+import { AddParticipantInEventMutation } from './mutations/AddParticipantInEventMutation';
 import { CreateEventMutation } from './mutations/CreateEventMutation';
 import { UpdateEventMutation } from './mutations/UpdateEventMutation';
 
 export const EventMutations = {
   CreateEventMutation,
   UpdateEventMutation,
-  AddUserInEventMutation,
+  AddParticipantInEventMutation,
```

Comment:
prefer FeatureActionField

feature -> the feature being write
action -> the action add, edit, delete, remove, update
field -> when is a field being changed or the secondary action

EventAddParticipant
EventCreate
EventUpdate

also, what is the differente between add, create and update? looks like you are following REST approaches

Reply:
AddParticipantInEvent is a mutation that create a participant and update an event

should I avoid it? and run them separately?

Reply:
instead i run:
- CreateParticipant
- UpdateEvent

i just run 
- AddParticipantInEvent